pkg/controller/provider/container/vsphere/watch.go

```diff
@@ -274,11 +274,7 @@ func (r *VMEventHandler) validate(vm *model.VM) (err error) {
 	}
 	err = policy.Agent.Submit(task)
 	if err != nil {
-		if errors.As(err, &policy.BacklogExceededError{}) {
```

Comment:
The `BacklogExceededError` no longer exists.  The channel is no longer limited because the `Task` is super small so the limit just causes trouble.